manifests/v1beta1/components/mysql/kustomization.yaml

```diff
@@ -2,8 +2,7 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-- mysql.yaml
-- pv.yaml
```

Comment:
Will we use the in-container FS to keep the db files?

Reply:
@gaocegege We mount PVC to the container FS here: https://github.com/kubeflow/katib/blob/bc7c1e19ee18e080b97b9ca0ed3d144c57a30c73/manifests/v1beta1/components/mysql/mysql.yaml#L68

It depends on the user's StorageClass, how it will store once container is deleted.